R/get_inboveg_layer_qualifiers.R

```diff
@@ -0,0 +1,123 @@
+#' @title Query layer qualifier information of recordings (relevé) from INBOVEG
+#'
+#' @description This function queries the INBOVEG database for layer
+#' qualifier information on recordings  for one or more surveys.
+#'
+#' @param survey_name A character string or a character vector, depending on
+#' multiple parameter, giving the name or names of the survey(s) for which you
+#' want to extract recordings information. If missing, all surveys are returned.
+#' @param connection dbconnection with the database 'Cydonia'
+#' on the inbo-sql07-prd server
+#' @param multiple If TRUE, survey_name can take a character vector with
+#' multiple survey names that must match exactly. If FALSE (the default),
+#' survey_name must be a single character string (one survey name) that can
+#' include wildcards to allow partial matches
+#'
+#' @return A dataframe with variables Name (of the survey), RecordingGivid (unique Id),
+#' UserReference, LayerCode, LayerDescription, QualifierCode, Qualifier Description,
+#' Elucidation and 'NotSure' in case the qualifier is doubtful, CoverCode and Cover percentage
+#'
+#' @importFrom glue glue_sql
+#' @importFrom DBI dbGetQuery
+#' @importFrom assertthat assert_that
+#'
+#' @export
+#' @family inboveg
+#' @examples
+#' \dontrun{
+#' library(inbodb)
+#' library(DBI)
+#' library(odbc)
+#' library(dplyr)
```

Comment:
Same remark here: as you are only using inbodb functions, loading other packages is not needed.  (I suppose we should also have a look at other functions to remove unnecessary packages in the examples.  Initially I was not aware of the fact that `dbDisconnect()` was added to inbodb, so I didn't question the fact that DBI and odbc were loaded in the examples...  dplyr on the other hand is needed in some of the examples.)
```suggestion
```